src/language-html/utils/content-security-policy-parser.js

```diff
@@ -0,0 +1,41 @@
+// Copied from https://github.com/helmetjs/content-security-policy-parser/blob/main/mod.ts
```

Comment:
```suggestion
/*
Based on https://github.com/helmetjs/content-security-policy-parser/blob/main/mod.ts with modifications:

1. Emit policy list instead of a `Map`, so we won't remove duplicated features.
1. Skip ASCII check, so we won't remove invalid features.
*/
```